docs/source/technicaldetails.rst

```diff
@@ -0,0 +1,12 @@
+Technical Details
+=================
+
+Connection Recycling
+---------------------
+Building the connection to the remote host is a computationally expensive operation, which is why lwreg caches the connection to the database as default behaviour.
+Whenever you work from lwreg connecting to the same database with just a single connection, making use of the caching is the most efficient.
+However, in a scenario where you are working with multiple connections to the same database, it is advised to disable the caching.
```

Comment:
```suggestion
However, in a scenario where you are working with multiple connections to the same database (for example if you have a lot of jobs that all use the database running), it is usually better to disable the caching in order to avoid exceeding the databases's connection limit.
```